mlrun/feature_store/retrieval/base.py

```diff
@@ -138,6 +138,9 @@ def _generate_vector(
     ):
         raise NotImplementedError("_generate_vector() operation not supported in class")
 
+    def unpersist_df(self, df):
```

Comment:
```suggestion
    def _unpersist_df(self, df):
```
Since it's an internal method meant to be overridden, better to start with `_`.